keras2onnx/_builtin.py

```diff
@@ -1473,6 +1473,36 @@ def convert_tf_select(scope, operator, container):
                                   op_version=9)
 
 
+@converter_func(TYPES.SelectV2)
+def convert_tf_selectv2(scope, operator, container):
+    if operator.target_opset < 9:
```

Comment:
Since oopb.add_node_with_output already specifies the opset, it is not needed to add an opset version check here and the converter framework will take care of it.